tee-worker/ts-tests/examples/direct-invocation/README.md

```diff
@@ -12,8 +12,8 @@ cd <path-to-tee-worker>
 2. run the ts demo
 
 ```
-cd <path-to-tee-worker/ts-tests/examples/direct-invocation>
-ts-node index.ts
+cd <path-to-tee-worker/ts-tests
```

Comment:
Do we need another '>' at the end?  like: `cd <path-to-tee-worker/ts-tests>`

Reply:
Thanks! 🙃